### PR TITLE
Re-add sap-hana to index.json

### PR DESCRIPTION
**Pull request by lagalbra**
It was removed in commit 43ed33d13552edc7d6b41ee141d9fd361234c28b due to using a feature not supported in CLI core version 2.0.67. The sap-hana extension was updated in https://github.com/Azure/azure-hanaonazure-cli-extension/pull/22 to cap the max CLI core version at 2.0.66 to avoid the breaking change. A follow-up change will be made once 2.0.67 is released to properly support it.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by lagalbra** [comment]
@limingu Let me know if this passes your min_profile checks; I have set the max CLI core version as you requested.

File: src/index.json [hunk 1]
```diff
@@ -1492,6 +1492,52 @@
                 "sha256Digest": "073d8dce3861e82f739586c652f67344f910ed853d17edea4ad86702a693b157"
             }
         ],
+        "sap-hana": [
+            {
+                "downloadUrl": "https://github.com/Azure/azure-hanaonazure-cli-extension/releases/download/0.3.8/sap_hana-0.3.8-py2.py3-none-any.whl",
+                "filename": "sap_hana-0.3.8-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.0.46",
```
Comment (limingu) [inline src/index.json]: You don't azext.maxCliCoreVersion here. That is not right.

**Comment by limingu** [comment]
> @limingu Let me know if this passes your min_profile checks; I have set the max CLI core version as you requested.

You don't ahve the maxCliCoreVersion in the metadata

**Comment by lagalbra** [comment]
> > @limingu Let me know if this passes your min_profile checks; I have set the max CLI core version as you requested.
> 
> You don't ahve the maxCliCoreVersion in the metadata

Done

File: src/index.json (hunk 1 above)
Comment (lagalbra) [reply src/index.json]: Done

**Comment by limingu** [comment]
The CI has error. https://travis-ci.org/Azure/azure-cli-extensions/jobs/544974546

**Comment by lagalbra** [comment]
I have removed my changes to the CI scripts to help debugging, so once the checks finish THIS time, I'm ready to have this merged.

**Pull request merged**